DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: tlf
 Type: Package
 Title: TLF Library
-Version: 1.5.0
+Version: 1.5.125.9000
```

Comment:
Hmmm we don't do this, the minor version should be set by AppVeyor.

Reply:
End users should always installe **released** versions anyways (those from https://github.com/Open-Systems-Pharmacology/TLF-Library/releases).
And if for some reason you need to install a dev version: I think, you would know what you have installed?
And if for some reason you don't know it anymore - just check if the version of the installed package is in the list of the released versions (https://github.com/Open-Systems-Pharmacology/TLF-Library/tags)

Some time ago we indeed used the X.Y.Z.9000 versioning - it's more headache than with the current approach.